.github/workflows/go.yml

```diff
@@ -31,5 +31,14 @@ jobs:
     - name: go vet
```

Comment:
Could you update README to explain how to use new sdk code ...?

Reply:
Done!